### ISSUE TITLE
requirements.txt needs titlecase.

### ISSUE THREAD

**Issue by benlk** [INN/cpd-settlements#1]
on a fresh installl and virtualenv: 

```
(cpd) $ tarbell serve
Traceback (most recent call last):
  File "/Users/blk/.virtualenvs/cpd/bin/tarbell", line 9, in <module>
    load_entry_point('tarbell==1.0.5', 'console_scripts', 'tarbell')()
  File "/Users/blk/.virtualenvs/cpd/src/tarbell/tarbell/cli.py", line 74, in main
    command.__call__(command, args)
  File "/Users/blk/.virtualenvs/cpd/src/tarbell/tarbell/cli.py", line 928, in __call__
    return self.fn(*args, **kw_args)
  File "/Users/blk/.virtualenvs/cpd/src/tarbell/tarbell/cli.py", line 424, in tarbell_serve
    with ensure_project(command, args) as site:
  File "/Users/blk/.virtualenvs/cpd/src/tarbell/tarbell/contextmanagers.py", line 58, in __enter__
    return self.ensure_site()
  File "/Users/blk/.virtualenvs/cpd/src/tarbell/tarbell/contextmanagers.py", line 79, in ensure_site
    site = TarbellSite(path)
  File "/Users/blk/.virtualenvs/cpd/src/tarbell/tarbell/app.py", line 277, in __init__
    self.project, self.base = self.load_project(path)
  File "/Users/blk/.virtualenvs/cpd/src/tarbell/tarbell/app.py", line 401, in load_project
    project = imp.load_module('project', filename, pathname, description)
  File "/Users/blk/tarbell/cpd_settlements/tarbell_config.py", line 43, in <module>
    from cpd_settlements import blueprint
  File "/Users/blk/tarbell/cpd_settlements/__init__.py", line 4, in <module>
    from titlecase import titlecase
ImportError: No module named titlecase
```

```
(cpd) $ pip install titlecase
You are using pip version 6.1.1, however version 9.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
Collecting titlecase
  Downloading titlecase-0.10.0-py2-none-any.whl
Installing collected packages: titlecase
Successfully installed titlecase-0.10.0
```

**Comment by benlk** [comment]
Related: Also needs ipdb at a certain version for some forms of error logging, since latest `pip install ipdb` fails on Pyhton 2.7.

**Comment by benlk** [comment]
Fixed in c0be826
